easybuild/tools/config.py

```diff
@@ -182,6 +182,14 @@
 EBPYTHONPREFIXES = 'EBPYTHONPREFIXES'
 PYTHON_SEARCH_PATH_TYPES = [PYTHONPATH, EBPYTHONPREFIXES]
 
+# options to handle header search paths in environment of modules
+MOD_SEARCH_PATH_HEADERS = {
+    "none": [],
```

Comment:
@lexming Do we actually have a valid use case for `none`?

I don't see how that can work, we really want people to pick between `CPATH` or `C_INCLUDE_PATH` & co, so I would remove the `none` option here...

Reply:
very reasonable :+1: